Autoprefixer.py

```diff
@@ -44,9 +44,11 @@ def run(self, edit):
 
 	def prefix(self, data):
 		try:
+			isCSS = splitext(basename(self.view.settings().get('syntax')))[0] == 'CSS'
```

Comment:
`splitext(basename(self.view.settings().get('syntax')))[0]` is used twice and could be put in a separate function that accepts a view and returns the syntax name.